Attendance_app.py

```diff
@@ -22,27 +22,46 @@
 manager_pass = 'ubuntu'
 
 current_class = 'Marauders'
-if current_class is not 'Marauders':
+if current_class != 'Marauders':
     current_class_obj = MainFile(current_class)
     FaceTrainObj = FaceTrain(current_class)
 else:
     current_class_obj = None
     FaceTrainObj = None
 
+"""Dynamic variables"""
+normal_width = 1920
+normal_height = 1080
 x, y = pyautogui.size()
 
+percentage_width = x/(normal_width/100)
+percentage_height = y/(normal_height/100)
+scale_factor = ((percentage_width + percentage_height)/2)/100
+
+title_fontsize = int(30 * scale_factor)
+minimum_size = 24
+if title_fontsize < minimum_size:
+    title_fontsize = minimum_size

```

Comment:
Prefer using [max()](https://docs.python.org/3/library/functions.html#max)

Reply:
I think max() function won't work here because there is no iterable to operate it on.